decidim-admin/app/views/decidim/admin/newsletters/index.html.erb

```diff
@@ -49,6 +49,10 @@
                   <%= icon_link_to "pencil", decidim_admin.edit_newsletter_path(newsletter), t("actions.edit", scope: "decidim.admin"), class: "edit" %>
                 <% end %>
 
+                <% if allowed_to?(:read, :newsletter, newsletter: newsletter) %>
```

Comment:
@andreslucena You really think we need to put this restriction? This does not affects user, is not something that can be abused and it can be useful to re-send newsletters for testing purposes. It seems a little bit artificial to add this restriction.

Reply:
Let me check it out with @decidim/product and I'll let you know

Reply:
After discussing this, we finally got at least one use case where this can be useful to send after the newsletter was already sent: to troubleshoot if an admin isn't in the segmented participants list and didn't receive the original email, and this email had any error that was reported by the participants.
Let's leave this as it originally was then (no restrictions to send the newsletter test if the newsletter was already sent)

Reply:
thanks @andreslucena !

Reply:
Thanks @microstudi  and @andreslucena.